includes/class-wp-job-manager-shortcodes.php

```diff
@@ -61,6 +61,8 @@ public function __construct() {
 		add_shortcode( 'job', [ $this, 'output_job' ] );
 		add_shortcode( 'job_summary', [ $this, 'output_job_summary' ] );
 		add_shortcode( 'job_apply', [ $this, 'output_job_apply' ] );
+
+		add_filter( 'paginate_links', [ $this, 'filter_paginate_links' ], 10, 1 );
```

Comment:
I wonder if this solution is too imprecise? For example, if there are other pagination links on this post, this will potentially change those links as well.

I wonder if it would be better to use the [`job_manager_pagination_args`](https://github.com/Automattic/WP-Job-Manager/blob/master/templates/pagination.php#L33) filter instead, and modify the `base` argument?

Reply:
Hey Alex! Thanks a lot for taking the time to review this PR!

Thanks for the `job_manager_pagination_args` filter suggestion. I tried to implement this approach, but the problem here is that the `paginate_links()` function adds the current URL args to the links; so even if we send in the 'base' argument a URL without the conflicting args, the `paginate_links()` adds them. The only filter I found we could hook to AFTER the conflicting args being re-added is the `paginate_links` one. 

Any other possible approaches? Let me know your thoughts!

Reply:
Ah thanks for the explanation. Makes sense.

I guess another approach would be to try to check the URL to see if it is the one we expect, but that seems like overkill. This is probably fine as-is, and we can iterate if needed.